### DISCUSSION TITLE
Synology PIA region help

### DISCUSSION THREAD

**Discussion by jchaps** [qdm12/gluetun#925] [Q&A] [answered]
I've been having difficulty connecting Private Internet Access to the Israel region, though it seems to work fine for other regions (e.g. CA Montreal).

Thinking I might need to update servers, I tried this command, but got an error.  Maybe the DNS server is cutting me off for spamming?

```
$ docker run -it --rm -v /volume1/docker/reshet/vpn:/gluetun qmcgaw/gluetun update -enduser -providers "private internet access" 

2022-04-04T03:50:52Z INFO merging by most recent 11193 hardcoded servers and 11193 servers read from /gluetun/servers.json
2022-04-04T03:50:53Z INFO updating Private Internet Access servers...
2022-04-04T03:50:56Z ERROR not enough servers found: 152 and expected at least 272
```

**Comment by qdm12** [comment, marked as answer]
Yeah I got the same thing, ranging from 150 to 230 servers...

Anyway I forced it to accept ~150 servers, and updated the server data for PIA in 9cdc9e9153d9ea1e825181529298088353a87b4f

You can therefore just `docker pull qmcgaw/gluetun` and that should do it. Next time it will not complain if you get at least 0.8 * 150 = 120 servers.
